### PR TITLE
Ng 16

### PR DESCRIPTION
**Pull request by anandtiwary**
## Description
Please include a summary of the change, motivation and context.

### Testing
Please describe the tests that you ran to verify your changes. Please summarize what did you test and what needs to be tested e.g. deployed and tested helm chart locally. 

### Checklist:
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,22 +21,22 @@
...
-    "@angular/router": "^15.2.9",
-    "@apollo/client": "^3.8.7",
+    "@angular/forms": "^16.1.8",
+    "@angular/material": "^16.1.7",
+    "@angular/platform-browser": "^16.1.8",
+    "@angular/platform-browser-dynamic": "^16.1.8",
+    "@angular/router": "^16.1.8",
+    "@apollo/client": "^3.8.5",
     "@fullstory/browser": "^1.7.1",
     "@hypertrace/hyperdash": "^1.2.1",
```
Comment (jake-bassett) [inline package.json:37]:
```suggestion
    "@hypertrace/hyperdash": "^1.3.0",
```

File: package.json [hunk 2]
```diff
@@ -22,22 +21,22 @@
...
-    "@hypertrace/hyperdash-angular": "4.0.0",
-    "@types/d3-hierarchy": "^2.0.5",
+    "@hypertrace/hyperdash-angular": "^5.1.0",
```
Comment (jake-bassett) [inline package.json:38]:
```suggestion
    "@hypertrace/hyperdash-angular": "^5.2.0",
```

File: .eslintrc.json [hunk 3]
```diff
@@ -181,11 +177,20 @@
         "no-useless-constructor": "off",
         "no-underscore-dangle": "off",
         "jsdoc/newline-after-description": "off",
+        "no-console": [
+          "error",
+          {
+            "allow": [
```
Comment (jake-bassett) [inline .eslintrc.json:183]: Are you sure we want this? We might be better off leaving it to loggerService.

Comment (anandtiwary) [reply .eslintrc.json:183]: This already exists. See above. I think the tool has re-arranged the rules.

File: package.json (hunk 2 above)
Comment (anandtiwary) [reply package.json:38]: can we do it separately? Just in case the new hd version leads to any UI issues.

File: .eslintrc.json (hunk 3 above)
Comment (jake-bassett) [reply .eslintrc.json:183]: Ah, fair enough.

File: package.json (hunk 2 above)
Comment (jake-bassett) [reply package.json:38]: Yeah, that's fine.

**Review by jake-bassett — DISMISSED** [review]

**Review by jake-bassett — APPROVED** [review]

**Pull request merged**